Content.Server.Database/Migrations/Sqlite/SqliteServerDbContextModelSnapshot.cs

```diff
@@ -559,7 +558,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .HasColumnType("TEXT")
                         .HasColumnName("hair_name");
 
-                    b.Property<JsonDocument>("Markings")
+                    b.Property<byte[]>("Markings")
```

Comment:
We should really fix this, I forgor if we had to manually update it.

Reply:
god dammit it happened again
@vulppine